src/CarbonAware.WebApi/src/CarbonAware.WebApi.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
```

Comment:
Can you add this file to `.gitignore`? So we won't commit it. Now it creates confusion.

Reply:
This should NOT be ignored.  It contains the c# documentation and if ignored, will make swagger and the auto openapi generation not have documentation.

Reply:
@KanishkT123 , @dnastrain . This file shouldn't be part of the commit, right?